client.js

```diff
@@ -450,6 +450,28 @@ LimitdClient.prototype.put = function (type, key, count, done) {
   return this._retriedRequest(request, done);
 };
 
+LimitdClient.prototype.get = function (type, key, done) {
+  const fireAndForget = typeof done !== 'function';
```

Comment:
Why would you call a getter as fire and forget?

Reply:
Removed the fire and forget option. I was following the pattern that the other methods used, but you're right that it doesn't make sense to call `get` that way.